ospray_modules/opendeck/ispc/cylindricStereoTracked/CylindricStereoTrackedCamera.ih

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2018, EPFL/Blue Brain Project
```

Comment:
no need for a ispc header file imo, just put the defintion to the ispc file itself.

Reply:
Having those headers don't hurt either, I'm going to leave that for now.

Reply:
It doesn't serve any purpose either :) One would not move private functions to headers as well if they are not used anywhere else.